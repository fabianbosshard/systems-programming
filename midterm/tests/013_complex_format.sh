exec "$1"  format="Fruit: @a - Count: @b" delim=" " <<EOF
Melon 8 3
Watermelon 4 9
EOF
