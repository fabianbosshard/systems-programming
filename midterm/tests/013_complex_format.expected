Fruit: Melon - Count: 8
Fruit: Watermelon - Count: 4
